### ISSUE TITLE
When I use Firefox, I should see the sprites.

### ISSUE THREAD

**Issue by ehgoodenough** [mocsarcade/enchiridion#10]
### Details

We are, for all intents and purposes, supporting primarily Chrome, mostly because we'll be bundling our code via Electron, which is really just Chromium. However, it would be nice if it worked (more or less) on most (or at least a few) other browsers. Such as Firefox.

When opening our game in Firefox, all our sprites are just boxes.
### Screenshot

<img width="535" alt="screen shot 2016-03-06 at 9 59 51 pm" src="https://cloud.githubusercontent.com/assets/1202938/13561878/c31ff3cc-e3e6-11e5-849d-37344ca2d1cd.png">
### Implementation

We're using `-webkit-mask`, which is (obviously) unique to Webkit. We need to either add declarations for other vendor-specific masks, or figure out why `mask` isn't working in React.

**Comment by Willamin** [comment]
Css masks only work with a vendor prefix in webkit browsers (Chrome, Safari, etc.) and in Firefox only work with inline svg, instead of a link to an image. (http://caniuse.com/#feat=css-masks)

**Comment by ehgoodenough** [comment]
Yeah, I think you're totally right. So we need to update the React entity to include the vendor-specific styles, not just the vanilla styles.

**Comment by ehgoodenough** [comment]
Either that, or switch from CSS Masks to CSS Color Rotates, or do the stupid lazy thing and color the sprites by canvas.

**Comment by Willamin** [comment]
My vote it to switch to the stupid lazy thing until Mozilla realizes it's 2016 and that they need to properly implement CSS masks. It looks like Webkit has had them since 2008 (https://webkit.org/blog/181/css-masks/). ¯_(ツ)_/¯
